### PR TITLE
en, zh: Update the warning for demonstration purposes

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

Update the warning for demonstration purposes to provide where to deploy for production environments. 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [x] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/get-started.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This document explains how to create a simple Kubernetes cluster and use it to d
 
 > **Warning:**
 >
-> These deployments are for demonstration purposes only. **Do not use** for qualification or in production!
+> These deployments are for demonstration purposes only. **Do not use** for qualification or in production! For production environments, see the instructions in **Deploy**>**Deploy TiDB Cluster**.
```
Comment (lilin90) [inline en/get-started.md:13]:
```suggestion
> These deployments are for demonstration purposes only. **Do not use** for qualification or in production! For production environments, see the instructions in **Deploy** > **Deploy TiDB Cluster**.
```

File: zh/get-started.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/get-started/','/docs-cn/dev/tidb-in-k
 
 > **警告：**
 >
-> 本文中部署例子仅用于测试目的。**不要**直接用于生产环境。
+> 本文中部署例子仅用于测试目的，**不要**直接用于生产环境。如果要在生产环境部署，请参阅**部署**>**部署 TiDB 集群**章节。
```
Comment (lilin90) [inline zh/get-started.md:13]:
```suggestion
> 本文中部署例子仅用于测试目的，**不要**直接用于生产环境。如果要在生产环境部署，请参阅**部署** > **部署 TiDB 集群**章节。
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Comment by qiancai** [comment]
After confirming with @DanielZhangQD, the TiDB Operator v1.0 is not maintained anymore so we can remove the needs-cherry-pick-1.0 label to skip the cherry-pick to v1.0.

/remove-label needs-cherry-pick-1.0

**Pull request merged**